app/src/main/java/com/allan/boardbuddies/Note.java

```diff
@@ -4,13 +4,22 @@
 public class Note {
     private String title;
     private String content;
+    private String name;
 
-    public Note(String title, String content){
+    public Note(String title, String content, String name){
```

Comment:
nit: it's not immediately clear what "name" means and how it differs from "title". Recommend "filename"